content/strategy-goals/strategy/search/product.md

```diff
@@ -20,69 +20,43 @@ Quicklinks:
 
 ## Where we are now
 
-Sourcegraph is the leading code intelligence platform in the market, used by tens of thousands of developers across enterprises of all sizes. The engine powering the platform is Code Search, which has advanced, novel search capabilities such as [structural search](https://learn.sourcegraph.com/how-to-use-structural-search-in-sourcegraph) and [diff/commit search](https://sourcegraph.com/notebooks/Tm90ZWJvb2s6MTI=).
+We are actively leveraging our prior search backend work to implement exciting new features like Lucky search, which make it possible to generate valid Sourcegraph queries form users' natural language input. Long term, this will help us build an intuitive search experience without making users learn our query language.
 
-The Search Product team has a dual mission:
+Over the next quarter, Search Product will focus on improving our core competency of code search, as well as extend the potential of the search platform. We're facing real competition from GitHub's new code search (currently in technology preview). Many customers and prospects find it good enough for their needs and there is real market pressure for us to differentiate quickly.
 
-- Increasing the value of the search as a platform for other teams to build upon (Code Insights, Batch Changes), with these other team's goals in mind.
-- Building differentiated user-facing products and features of its own.
-
-In addition to the core search experience, the Search Product team's scope includes [Code monitors](https://docs.sourcegraph.com/code_monitoring) and [Notebooks](https://sourcegraph.com/notebooks?tab=explore). Today, all products and features in Search Product's scope are included in every Sourcegraph contract.
-
-Over the past year, we’ve spent a significant amount of time cleaning up and improving the search backend. These efforts include, but are not limited to:
-
-- Implementing streaming across backends to minimize latency and memory usage
-- Partitioning search code into different domains and stages to manage complexity
-- Simplifying and unifying search result types
-
-Having just launched [Notebooks](https://docs.sourcegraph.com/notebooks) at the end of Q1, we anticipate to actively engage with new customers who purchased Sourcegraph because of them. We plan to take advantage of customer enthusiasm to create tight feedback loops between them and the team.
-
-We will also actively promote our experimental project, Compute, internally to Sales and Marketing and partner with them to engage with customers and get early product direction from the market.
+Our Q3 work will include paring back the search experience to its essentials and improving ergonomics via query language improvements. Simultaneously, the team will be working toward exposing metadata in search, eventually allowing us to search data other than code via Sourcegraph.
```

Comment:
Where can I read more about metadata, and how this relates to @ryankscott 's work?
I'm interested in this, there's even a draft PR about this here: https://github.com/sourcegraph/sourcegraph/pull/20932

Reply:
@malomarrec sorry for the late reply. Roadmap item with more info is here: https://github.com/sourcegraph/sourcegraph/issues/38366.

I know @camdencheek has also been in contact with @ryankscott and team about this work, so they either are already, or are getting aligned.